scripts/run_parsers.py

```diff
@@ -60,13 +60,46 @@ def clone_repository(self):
         #return git.Repo(self.repository_path)
         return git.Repo.clone_from(self.repository_url, self.repository_path)
 
+    def get_version_from_version_h(self, version_path: Path) -> Tuple[int, int, int] | None:
+        with open(file=version_path, mode='r') as version_file:
+            content = version_file.read()
+            match = re.search(r'#define\s+FW_MAJOR\s+(\d+)', content)
+            major = int(match.group(1))
+            match = re.search(r'#define\s+FW_MINOR\s+(\d+)', content)
+            minor = int(match.group(1))
+            match = re.search(r'#define\s+FW_PATCH\s+(\d+)', content)
+            patch = int(match.group(1))
+            return major, minor, patch
+
+    def tag_latest_versions(self):
+        # Creates a tag pointing to master for each vehicle type
+        # The version is extracted from version.h in the vehicle folder
+        # And a tag "VehicleType-Major.Minor.Patch" is created
+        for vehicle_type in self.valid_name_map.values():
+            version_path = Path(f'{self.repository_path}/{vehicle_type}/version.h')
+            version_path2 = Path(f'{self.repository_path}/Ardu{vehicle_type}/version.h')
+            path = version_path if version_path.exists() else version_path2
+            version = self.get_version_from_version_h(path)
+            if not version:
+                continue
+            major, minor, patch = version
+            try:
+                if vehicle_type == 'Sub':
+                    self.repository.create_tag(f'Ardu{vehicle_type}-{major}.{minor}.{patch}')
+                else:
+                    self.repository.create_tag(f'{vehicle_type}-{major}.{minor}.{patch}')
```

Comment:
I think it's better to just create the base tag, and add `'Sub': 'Sub'` to the `valid_name_map` (instead of having a special "Ardu" case for it).

That simplifies the tagging code, and future-proofs the parsing for if/when ArduSub ends up getting included in the combined ArduPilot releases.